specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2022-10/schemaregistry.json

```diff
@@ -0,0 +1,615 @@
+                "description": "URL location of schema, identified by schema group, schema name, and version."
+              },
+              "Schema-Id": {
+                "type": "string",
+                "description": "References specific schema in registry namespace."
+              },
+              "Schema-Id-Location": {
+                "type": "string",
+                "description": "URL location of schema, identified by schema ID."
+              },
+              "Schema-Group-Name": {
+                "type": "string",
+                "description": "References schema group."
+              },
+              "Schema-Name": {
+                "type": "string",
+                "description": "References schema name."
+              },
+              "Schema-Version": {
+                "type": "integer",
+                "description": "Version of the returned schema."
+              }
+            }
+          },
+          "415": {
+            "description": "Unsupported Media Type - Content type of the supplied schema is unsupported by schema group.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "An error response received from the Azure Schema Registry service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SchemaId": {
+      "type": "object",
+      "description": "Object received from the registry containing schema identifiers.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Schema ID that uniquely identifies a schema in the registry namespace."
+        }
+      }
+    },
+    "SchemaGroups": {
+      "type": "object",
+      "description": "Array received from the registry containing the list of schema groups.",
+      "properties": {
+        "schemaGroups": {
+          "type": "array",
+          "description": "Array of schema groups.",
+          "items": {
+            "$ref": "#/definitions/SchemaGroup"
+          }
+        }
+      }
+    },
+    "SchemaGroup": {
+      "type": "string",
+      "description": "Name of individual schema group in list of schema groups."
+    },
+    "SchemaVersions": {
+      "type": "object",
+      "description": "Array received from the registry containing the list of versions for specific schema.",
+      "properties": {
+        "schemaVersions": {
```

Comment:
Your property name must not be the same as the model name. This will break most SDK languages when generating code. At the very least, add an `x-ms-client-name` to the model name or schema name so that all client libraries generate the same.